src/lib.rs

```diff
@@ -110,9 +110,10 @@ pub struct StreamConfig {
     /// the audio buffers. For other drivers, only the number of requested channels is used, and
     /// order does not matter.
     pub channels: ChannelMap32,
-    /// Range of preferential buffer sizes. The library will make a bast-effort attempt at
-    /// honoring this setting, and in future versions may provide additional buffering to ensure
-    /// it, but for now you should not make assumptions on buffer sizes based on this setting.
+    /// Range of preferential buffer sizes, in units of audio samples per channel.
+    /// The library will make a bast-effort attempt at honoring this setting, and in future versions
```

Comment:
```suggestion
    /// The library will make a best-effort attempt at honoring this setting, and in future versions
```

Looks like I've made a typo writing this, if you could include the fix here that would be perfect!